zephyr/include/rtos/cache.h

```diff
@@ -8,12 +8,59 @@
 
 /* TODO: align with Zephyr generic cache API when ready */
 #define __SOF_LIB_CACHE_H__
+
+#if !CONFIG_ARM64
 #include <arch/lib/cache.h>
+#else
+#include <zephyr/cache.h>
+#endif /* CONFIG_ARM64 */
```

Comment:
This is a very welcome change. FYI  to @nashif  -- there's a change in progress on Zephyr side to allow us to move to sys_cache also on xtensa -> https://github.com/zephyrproject-rtos/zephyr/pull/50136

Reply:
yes, this is almost ready to go, we just fixed the inlining issue, ideally we shall be able to just use the zephyr cache API for everyone avoid indirection.